### PR TITLE
Migrate view-payments

### PR DESCRIPTION
**Pull request by jerekshoe**
## Description
Migrates the view-payments table from the `Pagination` react component to the `va-pagination` web component 

## Original issue(s)
department-of-veterans-affairs/va.gov-team#40323

## Acceptance criteria
- [x] view-payments table should be using `va-pagination` web component

## Definition of done
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)

### CODE REVIEW

**Review by cohnjesse — APPROVED** [review]
LGTM. I wonder if we shouldn't clean the logic in these components up some int he future?

File: src/applications/disability-benefits/view-payments/README.md [hunk 1]
```diff
@@ -29,3 +29,192 @@ When we render payments for the Veteran we do it using the `<Payments />` compon
 
 ## The back end code
 When the front end calls [/profile/payment_history](https://github.com/department-of-veterans-affairs/vets-api/blob/be3a39e6054afd8c6a2cbb61cfd215421a8a718a/app/controllers/v0/profile/payment_history_controller.rb#L1) in the back end we call the `new` method from the BGS People Service and assign the result, which is the Veteran's VA file number, to a new variable called `person`. We then use that `person` variable to call the `new` method from the BGS Payment Service and assign the result, a list of payments for the Veteran, to a new array of objects called `response`. We then send that response back to the front end using the `VetPaymentHistorySerializer`.
+
+## Mocking the `profile/payment_history` endpoint
+When developing locally, vets-api is not able to properly access BGS to retreive payment data.
```
Comment (scottymeyers) [inline src/applications/disability-benefits/view-payments/README.md:34]: retrieve*

Comment (jerekshoe) [reply src/applications/disability-benefits/view-payments/README.md]: Haha good catch. I've been seeing the word received so much lately that my wires got crossed

Comment (scottymeyers) [reply src/applications/disability-benefits/view-payments/README.md]: I also flip those myself so I know to look!

**Review by scottymeyers — APPROVED** [review]

**Pull request merged**